adapters/adview/adviewtest/params/race/banner.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
The params files are no longer necessary. We apologize, as the documentation update is still in progress. Please delete `/params/race/banner.json`, `/params/race/native.json`, and `/params/race/video.json`.

Reply:
ok, removed.